content/posts/December-2020.md

```diff
@@ -1,7 +1,7 @@
 ---
 title: "December 2020"
 date: 2020-12-04
-draft: true
+draft: false
```

Comment:
Oh my, I spent so much time trying to figure out why the post doesn't appear and gave up eventually.
Thanks!